docs/app/components/side-navigation/side-navigation.component.html

```diff
@@ -5,7 +5,7 @@
                 [ngModel]="versionService.version | async"
                 (ngModelChange)="versionService.setVersion($event)"
                 [btnRadio]="Version.AngularJS">AngularJS</button>
-            <button type="button" class="btn button-toggle-accent"
+            <button type="button" class="btn button-toggle-accent primary-override"
```

Comment:
Dont think this class is needed here as you should be able to override styles by doing the following:

```css
.side-navigation-container .side-navigation .btn-group .button-toggle-accent {
    // styles go here
}
```

Reply:
Sorry I meant to remove this.  I have overridden the styles in ux-aspects-microfocus like you suggested.